### PR TITLE
Improve `EJSON` package

### PR DESCRIPTION
**Pull request by filipenevola**
Migrated from #10422

### CODE REVIEW

**Comment by filipenevola** [comment]
@lamhieu-vk let's see if CircleCI will work fine here.

File: packages/ejson/package.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ Package.onUse(function onUse(api) {
 });
 
 Package.onTest(function onTest(api) {
-  api.use(['ecmascript', 'tinytest']);
+  api.use(['ecmascript', 'tinytest', 'mongo']);
```
Comment (sebakerckhof) [inline packages/ejson/package.js]: EJSON should not have a dependency on mongo IMO

**Comment by lamhieu-vk** [comment]
@filipenevola it was fined here and ready to merge! thank you very much.

**Comment by filipenevola** [comment]
I just published `ejson` package version 1.1.1 https://github.com/meteor/meteor/blob/devel/packages/ejson/CHANGELOG.md#v111-2019-12-20

**Pull request merged**